grails-gsp/core/src/main/groovy/org/grails/gsp/compiler/GroovyPageCompiler.groovy

```diff
@@ -225,11 +224,11 @@ class GroovyPageCompiler {
                 gpp.generateGsp(gsptarget)
                 gsptarget.flush()
                 // write static html parts to data file (read from classpath at runtime)
-                File htmlDataFile = new File(new File(targetDir, packageDir),  className + GroovyPageMetaInfo.HTML_DATA_POSTFIX)
+                File htmlDataFile = new File(new File(targetDir, packageDir), className + GroovyPageMetaInfo.HTML_DATA_POSTFIX)
```

Comment:
Seems redundant to create two `new File(targetDir, packageDir)` here and in line 231

Reply:
good call out, I have a change local to fix this but given closeness to release its part of some larger changes. can likely wait